www/havm10/StaticPage/home.html

```diff
@@ -32,49 +32,56 @@ <h2>Připoj se s námi k síti</h2>
     </div>
     <div class="content">
         <div class="left">
-            <h2>Seznam Akcí</h2>
+            <h2>Týdenní seznam akcí</h2>
             <ul>
                 <li>
                     <article class="action">
                         <h3>
-                            <a href="https://www.facebook.com/events/558278014612147/" target="blank">23.10.2018
-                                (18:00) -
-                                Základy Photoshopu vol. 2</a>
+                            <a href="https://www.facebook.com/events/279465039566064/" target="blank">30.10.2018
```

Comment:
_blank

Reply:
doplněno, moje chyba, nechal jsem se zlákat, že to bez podtržítka aspoň nějak fungovalo